pam/integration-tests/testdata/TestCLIAuthenticate/golden/warns_on_unsupported_arguments

```diff
@@ -0,0 +1,99 @@
+> ./pam_authd login socket=${AUTHD_TESTS_CLI_AUTHENTICATE_TESTS_SOCK} invalid_flag=foo bar
```

Comment:
Leftover file from the previous test

Reply:
Oh, damn... Yeah, it wouldn't have happened if I'd had rebased as I would have dropped the commit at all, but thanks for spotting it!